src/http/tx-scheduler/Transactions.ts

```diff
@@ -0,0 +1,162 @@
+/**
+ * Class that enables interaction with the TxScheduler ('/transactions')
+ */
+
+import { HttpClient } from '../client/http'
+import {
+  IDeployContractRequest,
+  IHttpGETRequest,
+  IHttpPOSTRequest,
+  IHttpResponse,
+  ISearchRequest,
+  ISendRawRequest,
+  ISendTransactionRequest,
+  ISendTransferRequest,
+  ITransactionResponse
+} from '../types'
+
+export class TxSchedulerTransactions {
```

Comment:
I don't like the name of this class haha. We can discuss a better name

Reply:
TxSchedulerClient??